### DISCUSSION TITLE
Still getting random auth errors

### DISCUSSION THREAD

**Discussion by ElZorroofOz** [bimmerconnected/bimmer_connected#698] [Q&A] [answered]
I poll status every 15 min with a shell loop calling the CLI (0.17.0).
Works for days, then suddenly the CLI returns status 120, sometimes in the middle of the night so I can't see it straight away.
Sometimes it comes back on status 0 by itself (I suspect server error), but sometimes not.

Re-doing a captcha (row, Australia) fixes it but is a PITB. the first status after a new captcha returns status 1 and a message "request mallformed", but still solves the problem.
I am not 100% sure they are auth and not quota errors.
Any similar issues?
What would you want me to capture to help pinpoint the issue and find a fix?

Thanks !

**Comment by rikroe** [comment, marked as answer]
Should be fixed with https://github.com/bimmerconnected/bimmer_connected/releases/tag/0.17.2 / #696.

**Reply by ElZorroofOz** [reply]
Thanks!  Have upgraded, will keep an eye an it.
